packages/components/src/internal/AppContexts.tsx

```diff
@@ -27,9 +28,11 @@ export const AppContexts: FC<Props> = props => {
             <AppContextProvider initialContext={initialAppContext}>
                 <SubNavContextProvider>
                     <NotificationsContextProvider>
-                        <Provider store={store}>
-                            <Router history={history}>{children}</Router>
-                        </Provider>
+                        <LabelPrintingProvider>
```

Comment:
LabelPrintingProvider does a call to getBarTenderConfigurationActionAPI. What would happen to apps without samplemanagement module? For example, freezermanager without samplemanagement (I think we have automated tests for it, though TC probably deploy all modules).

Reply:
Good call. I'll give it a try and see if this causes problem.

Reply:
I tried this and the "FM only" scenario was making an API call to the sampleManager controller that would fail silently, so it wasn't causing issues in the app (just extra noise in the JS console). To fix this, I left in the AppContexts usage of the LabelPrintingProvider, but added a check within that for `isSampleManagerEnabled()`. See https://github.com/LabKey/labkey-ui-components/pull/937/commits/d5f618d4249e117f4f9b1c9d6383d149b4ef5f30